buildout_1.rst

```diff
@@ -342,7 +330,13 @@ Be McGuyver
 
         $ ./bin/buildout -v
 
-    If strange egg versions are requested, check the dependencies declaration of your eggs and your version pinnings.
+    If strange egg versions are requested, check the dependencies declaration of your eggs and your version pinnings.  Here is an invaluable shell command that allows you to find all packages that depend on a particular egg and version:
+
+    .. code-block:: bash
+
+        $ grep your.egg.name.here /home/vagrant/buildout-cache/eggs/*.egg/EGG-INFO/requires.txt
```

Comment:
This information is very helpful, but using an absolute path only valid for the training makes it slightly less helpful.